### PR TITLE
Switchblade revival

### PR DESCRIPTION
**Pull request by Doges**
That's it, we are working on /next

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:16
+FROM node:17
```
Comment (almeidx) [inline Dockerfile:1]: Why use a version that won't become LTS? Might as well go with 18, which *will* become LTS in like 2 weeks

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     ]
   },
   "engines": {
-    "node": "> 14.0.0"
+    "node": "> 15.0.0"
```
Comment (almeidx) [inline package.json:19]: 🤔

**Review by almeidx — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Doges) [reply Dockerfile:1]: it breaks on 18

Comment (almeidx) [reply Dockerfile:1]: Can always just keep it on 16 😄

**Review by nitrog0d — APPROVED** [review]

**Review by almeidx — APPROVED** [review]

**Review by jpsl00 — APPROVED** [review]

**Pull request merged**